envs/staging-preprod/.envrc

```diff
@@ -2,7 +2,7 @@ export CHAIN_ID=1
 export THRESHOLD_NUMERATOR=2
 export THRESHOLD_DENOMINATOR=3
 export GENESIS_COMMITTEE_UTXO="4e65eddc6132c1b9891d43f9d67a4a0a7b6e614b648f59dc97538f3f23eb96bf#1"
-# cardano-cli address key-hash --payment-verification-key-file <path to authority vkey>
+# cardano-cli latest address key-hash --payment-verification-key-file <path to authority vkey>
```

Comment:
And here?

Reply:
yes, this is a "note" to show how to generate the governance authority - we should leave it